style.md

```diff
@@ -151,6 +152,65 @@ An interface is two fields:
 If you want interface methods to modify the underlying data, you must use a
 pointer.
 
+### Verify Interface Compliance
+
+For types that are known or required to implement specific interfaces, add
+checks to verify interface compliance at compile time.
+
+<table>
+<thead><tr><th>Bad</th><th>Good</th></tr></thead>
+<tbody>
+<tr><td>
+
+```go
+type Handler struct{ /* ... */ }
```

Comment:
ultra style nit: either break this into multiple lines (as it would be defined outside of an example) or add a space before the opening brace (here and below for the adjacent `Handler` definition)